### DISCUSSION TITLE
我的耕种奖励目标地址已经收到0.25xch，但是钱包没有收到奖励（钱包地址已同步）

### DISCUSSION THREAD

**Discussion by flynesses** [Chia-Network/chia-blockchain#8187] [Q&A]
我的耕种奖励目标地址已经收到0.25xch，但是钱包没有收到奖励（钱包地址已同步）
![image](https://user-images.githubusercontent.com/39182553/130557067-506d2dc8-b8ac-42df-93b6-809dcca08712.png)
![image](https://user-images.githubusercontent.com/39182553/130557111-32e27c24-347c-496b-a630-7b03d246cca8.png)

**Comment by sixym3** [comment]
Same here, it says that I farmed a block while pooling. But in my transaction history I did not get the 0.25 xch at the height where the pool says I should have won it. My wallet is synced and the chia blockchain finder payout address matches the rewards target address.

**Reply by supercating** [reply]
我也一样，并且我在池中找到了0.25，但是没有在钱包上收到；
